tests/unit/test_graph.py

```diff
@@ -168,6 +185,15 @@ def test_no_direction(community_with_trustlines):
     assert path == []
 
 
+def test_valid_path_raise_ValueError(complex_community_with_trustlines_and_fees):
```

Comment:
Did you mean test_valid_path_raises_no_value_error? 
Also please dont use CamelCase in functions in python. 
If you write a test for a previous bug you should write a comment with the issue number or describe the bug here. Otherwise this test might get deleted in the future because no one know why you test something random that should work.

Reply:
I did mean "test_valid_path_raise_ValueError" , it tests whether a valid path raises a ValueError and does not assert anything on the result (but I changed it if you like it). It was CamelCase because it refers to the name of the error.
Also, I wrote the test to show that the bug was corrected, and do not mind if it gets deleted in the future.

Reply:
Well I would mind. It is now a regression test and should stay.